### PR TITLE
Build the homepage

### PR DESCRIPTION
**Pull request by anthonydillon**
Built the homepage from the [latest copy](https://docs.google.com/document/d/1vn6it-3px_KbplnajPlWPonU6W_xp4JdhE02kNQouZA/edit#).

## QA
- Open the demo
- Check the copy matches the copy doc

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1403

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @anthonydillon. Can we use the light blue for the strips please, as on jaas.ai.

I would suggest some have the dark blue background, but I can't see from the doc that any strip is more important than another.

If we have illustrations, we could alternate their position left and right on the strips.

Thanks.

**Comment by spencerbygraves** [comment]
@anthonydillon should "Windows | Mac | Linux" in the top section be links to the bottom section?

**Comment by anthonydillon** [comment]
> Thanks @anthonydillon. Can we use the light blue for the strip backgrounds please, as on jaas.ai.

Done.

> I would suggest some have the dark blue background, but I can't see from the doc that any strip is more important than another.

The only dark one I have is the black strip but if you point me to an existing one I'll bring it over.

> If we have illustrations, we could alternate their position left and right on the strips.

Of course, left it basic for now.

**Comment by anthonydillon** [comment]
> should "Windows | Mac | Linux" in the top section be links to the bottom section?

Not sure, maybe it should just be logos with links to the bottom like you said

**Comment by spencerbygraves** [comment]
@anthonydillon in the bottom OLM section you've used a Linux logo, but it's an Ubuntu one in the doc.

File: templates/index.html [hunk 1]
```diff
@@ -1,455 +1,493 @@
...
+						Simplify global operations and change management. The Juju OLM provides standardized, schema-driven YAML configuration for each operator, with consistent semantics for config updates, reads, and rollouts.
+					</p>
+				</div>
+			</div>
+		</section>
+
+		<section class="p-strip--muted-suru">
+			<div class="row">
+				<div class="col-8">
+					<h2>Day 1, day 2, and beyond</h2>
```
Comment (pmahnke) [inline templates/index.html:337]: remove the comma after 'day 2'

File: templates/index.html [hunk 2]
```diff
@@ -1,455 +1,493 @@
...
+		</section>
+
+		<section class="p-strip--muted-suru">
+			<div class="row">
+				<div class="col-8">
+					<h2>Multi-arch operators</h2>
+					<p class="p-heading--four">ARM, x86, POWER, Z are all supported
+					</p>
+					<p>
+						Integrate heterogeneous environments transparently &mdash; connect services on the mainframe to applications on the cloud and in the data center.
```
Comment (pmahnke) [inline templates/index.html:393]: centre not center

**Review by pmahnke — CHANGES_REQUESTED** [review]
Comments inline

**Comment by spencerbygraves** [comment]
@anthonydillon could you try this svg for the logos at the top please. Thanks.

https://assets.ubuntu.com/v1/9b2f65f0-juju.is-logos-banner.svg

**Comment by anthonydillon** [comment]
> @anthonydillon could you try this svg for the logos at the top please. Thanks.

Done/

**Comment by anthonydillon** [comment]
@spencerbygraves @pmahnke This one is ready for another look

File: templates/index.html [hunk 3]
```diff
@@ -1,455 +1,474 @@
...
-
-{% endblock %}
+	<section class="p-strip--suru-bottom has-accent is-dark" style="background-size: auto 100%, cover; background-position: right center; background-image: url('https://assets.ubuntu.com/v1/a9dab044-suru-right.svg'), linear-gradient(266deg, #044f66, #022935)">
+		<div class="row u-equal-height">
+			<div class="col-6">
+				<h1>Universal Operator Lifecycle Manager</h1>
+				<p>Better operators for enterprise apps with a full application graph and declarative integration for both Kubernetes and legacy estate.</p>
+				<p>
+					<a href="/tutorials" class="p-button--positive">Get started</a>
+					<a href="/tour" class="p-link--inverted">Take the tour&nbsp;&rsaquo;</a>
```
Comment (pmahnke) [inline templates/index.html]: remove the take the tour link from here and the nav

**Review by pmahnke — CHANGES_REQUESTED** [review]
only the one comment

**Review by pmahnke — APPROVED** [review]
+1

**Pull request merged**